packages/web-react/src/components/Tooltip/demo/index.tsx

```diff
@@ -7,37 +7,37 @@ import ReactDOM from 'react-dom/client';
 import icons from '@lmc-eu/spirit-icons/icons';
 import DocsSection from '../../../../docs/DocsSections';
 import { IconsProvider } from '../../../context';
-import TooltipClickable from './TooltipClickable';
+import TooltipPlacements from './TooltipPlacements';
 import TooltipDefault from './TooltipDefault';
 import TooltipDismissible from './TooltipDismissible';
 import TooltipDismissibleViaJS from './TooltipDismissibleViaJS';
-import TooltipFloatingUI from './TooltipFloatingUi';
-import TooltipOnHover from './TooltipOnHover';
-import TooltipPlacements from './TooltipPlacements';
+import TooltipHover from './TooltipHover';
+import TooltipIcon from './TooltipIcon';
+import TooltipWithFloatingUI from './TooltipWithFloatingUI';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
     <IconsProvider value={icons}>
       <DocsSection title="Placements" stackAlignment="stretch">
         <TooltipPlacements />
       </DocsSection>
-      <DocsSection title="Static Tooltip (No Interaction)">
+      <DocsSection title="Default">
         <TooltipDefault />
       </DocsSection>
-      <DocsSection title="Tooltip on Hover (Pure CSS)" stackAlignment="stretch">
-        <TooltipOnHover />
-      </DocsSection>
-      <DocsSection title="Tooltip on Click (JavaScript)">
-        <TooltipClickable />
-      </DocsSection>
       <DocsSection title="Dismissible Tooltip">
         <TooltipDismissible />
       </DocsSection>
       <DocsSection title="Dismissible Tooltip via JS API">
         <TooltipDismissibleViaJS />
       </DocsSection>
-      <DocsSection title="Advanced Positioning">
-        <TooltipFloatingUI />
+      <DocsSection title="Tooltip Only on Hover">
+        <TooltipHover />
+      </DocsSection>
+      <DocsSection title="Tooltip on Icon Component">
+        <TooltipIcon />
+      </DocsSection>
+      <DocsSection title="Advanced Floating Functionality" stackAlignment="stretch">
+        <TooltipWithFloatingUI />
```

Comment:
What about renaming this demo to `TooltipAdvancedFloating`?

Reply:
good idea, renamed to `TooltipAdvancedFloating` to better match demo title